release.yaml

```diff
@@ -105,10 +106,12 @@ rancher-monitoring:
   - 104.0.0-rc1+up45.31.1
   - 103.0.3+up45.31.1
   - 102.0.3+up40.1.2
+  - 103.0.4+up45.31.1
 rancher-monitoring-crd:
   - 104.0.0-rc1+up45.31.1
   - 102.0.3+up40.1.2
   - 103.0.3+up45.31.1
+  - 103.0.4+up45.31.1
```

Comment:
I don't think we should be making changes to `rancher-monitoring` and `rancher-monitoring-crd` in this PR...

Reply:
Just rebased the branch and it's fixed.